### PR TITLE
Fix sticky footer for IE

### PR DESCRIPTION
**Pull request by AJEllender**
Fixes Sticky Footer for IE versions 10+

### CODE REVIEW

File: footer.php [hunk 1]
```diff
@@ -5,7 +5,7 @@
   </div>
 <?php endif; ?>
 
-<footer>
+<footer style="height: 30px; background-color: purple;">
```
Comment (danimalweb) [inline footer.php]: Do we need these inline styles?

Comment (mrchimp) [inline footer.php]: nope

**Comment by danimalweb** [comment]
Can you test that your code change works with this scenario:

https://github.com/lambdacreatives/tofino/issues/75

Thanks.

**Comment by AJEllender** [comment]
Yep. Just Tested in IE10, Edge13, Chrome and Safari. 

All works as expected with too much and too little page content.

**Comment by danimalweb** [comment]
I merged this and then reverted it. Can you take another look? We already have some code in header.php and footer.php checking for footer_sticky with the class name 'wrapper'. Do we just need to update the class and move the position of the wrapper?

**Comment by AJEllender** [comment]
It's not quite that simple. We either need 2 wrappers (because in IE, height and min-height don't work for `flex-direction: column` elements unless the element is itself a child of a flex element), or the body needs to get a class that sets it's display property to flex. Either are fine, although I generally try and preserve the body tag with as few extra modifications as possible given that it's already being used to house a number of other classes for Wordpress.

**Comment by danimalweb** [comment]
I'd prefer just the one wrapper element if possible with whatever adjustments need to be made to the body class. I'm already hooking in a bunch of extras into the body tag in strucutre.scss.

**Pull request merged**